### DISCUSSION TITLE
Editing the visual part

### DISCUSSION THREAD

**Discussion by devopg** [imzbf/md-editor-v3#260] [Ideas]
Just to ask. Is it realistic to add a new feature so that you can not only edit the markup, but also edit the visualization?
Or is it necessary to change almost everything?

![image](https://user-images.githubusercontent.com/61819835/236292492-38ba42d3-915a-4baa-a868-a8f662309023.png)

**Comment by imzbf** [comment]
I'm sorry, this feature cannot be achieved currently, because there is no good mapping relationship. 

This is the foundation of editors like `Typora`. It's awesome but no time to study its principle. If there is time later, I may try it out.
